domains/game/Game.jsx

```diff
@@ -201,6 +225,44 @@ const Game = ({ playerName, game }) => {
                     <Text>Replace pucks</Text>
                 </TouchableOpacity>
             ) : null}
+
+            <Modal
+                animationType="slide"
+                visible={modalVisible}
+                transparent={true}
+                onRequestClose={() => {
+                    setModalVisible(!modalVisible);
+                }}
+            >
+                <View style={styles.modalView}>
+                    <Pressable
+                        style={[styles.button, styles.buttonClose]}
+                        onPress={() => closeAdvice()}
+                    >
+                        <Text>
+                            {bestMove
+                                ? bestMove
+                                : "Bot is thinking about the best move..."}
+                        </Text>
+                    </Pressable>
+                </View>
+            </Modal>
+            {game.currentPlayer == playerName && game.remainingTurns > 0 ? (
+                <TouchableOpacity
+                    style={{
+                        position: "absolute",
+                        top: 10,
+                        right: 10,
+                        zIndex: 999,
+                        backgroundColor: "white",
+                    }}
+                    onPress={() => showAdvice()}
```

Comment:
you can pass showAdvice directly, now need to create a function that does nothing more